src/components/Firmware/V3UploadLocalFirmware.tsx

```
@@ -131,10 +131,7 @@ const V3UploadLocalFirmware: React.FC = () => {
         {intl.formatMessage({ id: 'TR_LOCAL_FIRMWARE_UPLOAD' }) ||
           '本地固件上传'}
       </h3>
-      <p className="text-sm text-gray-500">
-        {intl.formatMessage({ id: 'TR_LOCAL_FIRMWARE_DESCRIPTION' }) ||
-          '选择本地固件文件进行更新。上传前请确保文件来源可靠。'}
-      </p>
+      
 
```

ESLint

[error] Delete `······⏎`

(prettier/prettier)